modules/persistent-storage-csi-efs-metrics-procedure-gui.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
```

Comment:
we should highlight that it is possible to disable metrics after enabling it. Set it back to "Disabled".
Customers may enable it and find that this is indeed to impactful, so it is important to explicitly mention it. I believe QE tested this.

Reply:
Done.